.github/workflows/docker-daily.yml

```diff
@@ -6,17 +6,49 @@ on:
   
 jobs:
 
-  daily:
-
+  daily_amd64:
+    name: Build docker image for amd64
     runs-on: ubuntu-latest
 
     steps:
     - uses: actions/checkout@master
       with:
         submodules: recursive
     - name: Build the Docker image
-      run: docker build . --file Dockerfile --tag ${{ secrets.DOCKER_REPO }}:daily
+      run: docker build . --file Dockerfile --tag ${{ secrets.DOCKER_REPO }}:daily-amd64
     - name: Log into docker hub
       run: docker login --username ${{ secrets.DOCKER_USERNAME }} --password ${{ secrets.DOCKER_PASSWORD }}
     - name: Push daily to Docker Hub
-      run: docker push ${{ secrets.DOCKER_REPO }}:daily
+      run: docker push ${{ secrets.DOCKER_REPO }}:daily-amd64
+
+  daily_arm64:
+    name: Build docker image for arm64
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+      with:
+        submodules: recursive
+    - uses: docker/setup-qemu-action@v1
+      name: Set up QEMU
+      with:
+        platforms: arm64
+    - uses: docker/setup-buildx-action@v1
+      name: Set up Docker Buildx
+    - name: Log into docker hub
+      run: docker login --username ${{ secrets.DOCKER_USERNAME }} --password ${{ secrets.DOCKER_PASSWORD }}
+    - name: Build and Push the Docker image
+      run: docker buildx build . --file Dockerfile.aarch64 --tag ${{ secrets.DOCKER_REPO }}:daily-arm64 --platform linux/arm64 --push
```

Comment:
Is there a way we can build arm64 with buildx? This means we could simplify these two with a matrix.